airbyte-integrations/connectors/source-bing-ads/metadata.yaml

```diff
@@ -16,7 +16,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 47f25999-dd5e-4636-8c39-e7cea2453331
-  dockerImageTag: 2.23.2
+  dockerImageTag: 3.0.0
```

Comment:
I'd love to make this a minor bump so it has no user impact, but I don't think that's possible with the existing QA checks